src/scythe-bidder/create-room.tsx

```diff
@@ -59,6 +63,19 @@ export default function CreateRoom({ onCreate }: { onCreate: () => void }) {
                   ))}
               </Select>
             </Form.Item>
+            <Form.Item>
+              <Select<string>
+                value={gameType}
+                onChange={(value) => {
+                  setGameType(value);
+                }}
```

Comment:
Here - if the user changes from IFA to base - could we check the value of `numPlayers` and set it to 5 if it's greater than 5? Would also be good to notify the user, maybe through the [Notification](https://ant.design/components/notification/) module.

Reply:
If we do this, we should also change line 57 so it doesn't allow selecting more than 5 players when IFA is not chosen.

Reply:
Will do.